### ISSUE TITLE
Automatically configure Packit projects to use OpenScanHub

### ISSUE THREAD

**Issue by siteshwar** [packit/packit-service#2630]
As of now, we are scanning only 73 packages through OpenScanHub integration with Packit. This issue tracks opening pull requests to [configure](https://packit.dev/posts/openscanhub-prototype#setup) projects to start submitting scans to OpenScanHub.

This [list](https://github.com/user-attachments/files/17715654/packit-osh-packages.txt) contains packages currently being scanned through OpenScanHub via Packit.

**Comment by siteshwar** [comment]
Add [polkit](https://github.com/polkit-org/polkit/pull/527) to the list.

**Comment by siteshwar** [comment]
Another possibility is to do differential scans with latest koji builds. Although the results may look confusing as they may contain findings from changes outside a pull request. So, it probably makes sense to do such scans before a package release.

**Comment by lbarcziova** [comment]
> Although the results may look confusing as they may contain findings from changes outside a pull request.

I agree this could be quite misleading and inaccurate, so I would probably avoid doing that.

> So, it probably makes sense to do such scans before a package release.

Can you elaborate on this a bit, please?

**Comment by siteshwar** [comment]
> > Although the results may look confusing as they may contain findings from changes outside a pull request.
> 
> I agree this could be quite misleading and inaccurate, so I would probably avoid doing that.
> 
> > So, it probably makes sense to do such scans before a package release.
> 
> Can you elaborate on this a bit, please?

We can do a differential scan on each commit to the target branch, but that could be too resource consuming and results may not be visible to the user. I was thinking about triggering a differential scan with one of [these jobs](https://packit.dev/docs/fedora-releases-guide#propose-downstream-or-pull-from-upstream), but I am not sure how we can make the results visible to the user. Otherwise, we may just have to rely on Packit as the Fedora CI to perform a differential scan.

**Comment by lbarcziova** [comment]
> I was thinking about triggering a differential scan with one of [these jobs](https://packit.dev/docs/fedora-releases-guide#propose-downstream-or-pull-from-upstream), but I am not sure how we can make the results visible to the user. Otherwise, we may just have to rely on Packit as the Fedora CI to perform a differential scan.

Ah I see, thanks for the explanation! Those are though triggered on upstream release, so doing that would have a similar effect as triggering directly on downstream pull requests.
